setup.py

```diff
@@ -58,6 +58,6 @@
     package_dir={'':'src'},
     package_data={'': ['*.xsd', '*.xml']},
     test_suite='metaswitch.crest.test',
-    install_requires=["pyzmq", "py-bcrypt", "cyclone==1.0", "cql", "lxml", "msgpack-python", "pure-sasl"],
+    install_requires=["pyzmq", "py-bcrypt", "cyclone==1.0", "cql", "lxml", "msgpack-python", "pure-sasl", "prctl"],
```

Comment:
The source tarball (admittedly also on pypi) does contain a BSD LICENSE file, as does this github repo of the code: https://github.com/slideinc/prctl

setproctitle has the wrong behaviour for what we want (it overwrites argv as well as calling prctl).

python-prctl depends on libcap-dev.  I can switch to that, but it annoyingly drags in new build- and run-time dependencies that we won't use.